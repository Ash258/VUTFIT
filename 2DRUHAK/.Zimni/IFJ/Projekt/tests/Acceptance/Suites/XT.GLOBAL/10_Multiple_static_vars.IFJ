' Multiple static variables

function counter(a as integer) as integer
	static c as integer = 25
	c = c + 1
	return c
end function

function multiplier(a as integer) as integer
	static c as integer = 2
	c = c * 2
	return c
end function

scope
	dim res as integer
	res = counter(25)
	print res;
	res = counter(0)
	print res;
	res = multiplier(2)
	print res;
	res = multiplier(0)
	print res;
	res = counter(0)
	print res;
	res = counter(0)
	print res;
	res = multiplier(0)
	print res;
end scope
