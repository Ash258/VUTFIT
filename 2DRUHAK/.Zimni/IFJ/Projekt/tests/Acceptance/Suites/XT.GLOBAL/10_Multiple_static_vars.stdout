 26 27 4 8 28 29 16